packages/server/tests/acceptance/htsPrecompile/tokenCreate.spec.ts

```diff
@@ -67,13 +67,13 @@ describe('@tokencreate HTS Precompile Token Create Acceptance Tests', async func
   const TOKEN_DECIMALS = BigInt(8);
 
   const accounts: AliasAccount[] = [];
-  let mainContractAddress;
+  let mainContractAddress: string;
   let HTSTokenContractAddress;
   let NftHTSTokenContractAddress;
   let NftSerialNumber;
   let HTSTokenContract;
   let NFTokenContract;
-  let mainContract;
+  let mainContract: ethers.BaseContract;
   let mainContractOwner;
   let mainContractReceiverWalletFirst;
   let mainContractReceiverWalletSecond;
```

Comment:
**nit:** can we specify the types for these, as well?

Reply:
Added, thanks